semantic_classes:
  # --- basic structure (important for Mapping) ---
  background:
    id: 0
    color: [0, 0, 0]
  floor:
    id: 1
    color: [128, 128, 128] 
  wall:
    id: 2
    color: [200, 200, 200] 
  ceiling:
    id: 3
    color: [220, 220, 220]
  door:
    id: 4
    color: [139, 69, 19]   
  window:
    id: 5
    color: [173, 216, 230] 

  # --- lifeforms (dynamic "objects") ---
  person:
    id: 6
    color: [255, 0, 0]     
  bicycle:
    id: 7
    color: [0, 255, 0]
  car:
    id: 8
    color: [0, 0, 255]
  motorcycle:
    id: 9
    color: [255, 255, 0]
  airplane:
    id: 10
    color: [0, 255, 255]
  bus:
    id: 11
    color: [255, 0, 255]
  train:
    id: 12
    color: [192, 192, 192]
  truck:
    id: 13
    color: [128, 0, 0]
  boat:
    id: 14
    color: [0, 128, 128]
  bird:
    id: 15
    color: [128, 128, 0]
  cat:
    id: 16
    color: [0, 0, 128]
  dog:
    id: 17
    color: [255, 165, 0]   
  horse:
    id: 18
    color: [255, 20, 147]
  sheep:
    id: 19
    color: [255, 105, 180]
  cow:
    id: 20
    color: [218, 112, 214]

  # --- objects (small) ---
  backpack:
    id: 21
    color: [238, 130, 238]
  umbrella:
    id: 22
    color: [75, 0, 130]
  handbag:
    id: 23
    color: [148, 0, 211]
  tie:
    id: 24
    color: [138, 43, 226]
  suitcase:
    id: 25
    color: [72, 61, 139]
  frisbee:
    id: 26
    color: [65, 105, 225]
  skis:
    id: 27
    color: [30, 144, 255]
  snowboard:
    id: 28
    color: [0, 191, 255]
  sports ball:
    id: 29
    color: [135, 206, 235]
  kite:
    id: 30
    color: [0, 255, 127]
  baseball bat:
    id: 31
    color: [50, 205, 50]
  baseball glove:
    id: 32
    color: [154, 205, 50]
  skateboard:
    id: 33
    color: [107, 142, 35]
  surfboard:
    id: 34
    color: [85, 107, 47]
  tennis racket:
    id: 35
    color: [255, 215, 0]
  bottle:
    id: 36
    color: [218, 165, 32]
  wine glass:
    id: 37
    color: [184, 134, 11]
  cup:
    id: 38
    color: [210, 105, 30]
  fork:
    id: 39
    color: [139, 69, 19]
  knife:
    id: 40
    color: [160, 82, 45]
  spoon:
    id: 41
    color: [165, 42, 42]
  bowl:
    id: 42
    color: [128, 0, 0]
  banana:
    id: 43
    color: [255, 228, 181]
  apple:
    id: 44
    color: [255, 99, 71]
  sandwich:
    id: 45
    color: [255, 127, 80]
  orange:
    id: 46
    color: [255, 140, 0]
  broccoli:
    id: 47
    color: [34, 139, 34]
  carrot:
    id: 48
    color: [233, 150, 122]
  hot dog:
    id: 49
    color: [205, 92, 92]
  pizza:
    id: 50
    color: [240, 128, 128]
  donut:
    id: 51
    color: [233, 150, 122]
  cake:
    id: 52
    color: [250, 128, 114]

  # --- furniture (Indoor) ---
  chair:
    id: 53
    color: [255, 20, 147] # Pink
  couch:
    id: 54
    color: [199, 21, 133]
  potted plant:
    id: 55
    color: [50, 205, 50]  # Gr√ºn
  bed:
    id: 56
    color: [219, 112, 147]
  dining table:
    id: 57
    color: [153, 50, 204]
  toilet:
    id: 58
    color: [240, 248, 255]
  tv:
    id: 59
    color: [70, 130, 180]
  laptop:
    id: 60
    color: [100, 149, 237]
  mouse:
    id: 61
    color: [123, 104, 238]
  remote:
    id: 62
    color: [106, 90, 205]
  keyboard:
    id: 63
    color: [72, 61, 139]
  cell phone:
    id: 64
    color: [65, 105, 225]
  microwave:
    id: 65
    color: [0, 0, 139]
  oven:
    id: 66
    color: [25, 25, 112]
  toaster:
    id: 67
    color: [139, 0, 139]
  sink:
    id: 68
    color: [128, 0, 128]
  refrigerator:
    id: 69
    color: [75, 0, 130]
  book:
    id: 70
    color: [255, 215, 0]
  clock:
    id: 71
    color: [218, 165, 32]
  vase:
    id: 72
    color: [184, 134, 11]
  scissors:
    id: 73
    color: [210, 105, 30]
  teddy bear:
    id: 74
    color: [160, 82, 45]
  hair drier:
    id: 75
    color: [139, 69, 19]
  toothbrush:
    id: 76
    color: [165, 42, 42]
  
  # --- EXTRA INDOOR objects (Warehouse/Office) ---
  box:
    id: 77
    color: [210, 180, 140] 
  trash can:
    id: 78
    color: [105, 105, 105]
  cabinet:
    id: 79
    color: [139, 69, 19]
  shelf:
    id: 80
    color: [205, 133, 63]
  pallet:
    id: 81
    color: [244, 164, 96]
  forklift:
    id: 82
    color: [255, 140, 0]
  rack:
    id: 83
    color: [192, 192, 192]
  conveyor belt:
    id: 84
    color: [47, 79, 79]
  monitor:
    id: 85
    color: [70, 130, 180]
  printer:
    id: 86
    color: [112, 128, 144]
  desk:
    id: 87
    color: [139, 69, 19]
  whiteboard:
    id: 88
    color: [255, 255, 240]
  projector:
    id: 89
    color: [220, 220, 220]
  fire extinguisher:
    id: 90
    color: [255, 0, 0]
  exit sign:
    id: 91
    color: [0, 255, 0]
  stairs:
    id: 92
    color: [255, 228, 196]
  railing:
    id: 93
    color: [169, 169, 169]
  column:
    id: 94
    color: [245, 245, 245]
  bench:
    id: 95
    color: [139, 90, 43]
  stool:
    id: 96
    color: [205, 133, 63]
  carpet:
    id: 97
    color: [128, 0, 0]
  curtain:
    id: 98
    color: [255, 228, 225]
  mirror:
    id: 99
    color: [176, 224, 230]
  robot:
    id: 100
    color: [0, 0, 255]  